File: src/test/java/org/cerion/stocklist/TestBase.java
package org.cerion.stocklist;

import org.junit.Assert;

import java.util.Date;

public class TestBase {

    public static void assertDateEquals(Date expected, Date actual) {
        Assert.assertEquals(expected.getTime() / 1000, actual.getTime() / 1000);
    }
}

File: src/test/java/org/cerion/stocklist/TestBase.kt
package org.cerion.stocklist

import org.junit.Assert
import org.junit.Assert.assertEquals
import java.util.*

open class TestBase {

    fun assertEqual(expected: Double, actual: Float, message: String) {
        assertEquals(message, expected, actual.toDouble(), 0.005)
    }

    fun assertDateEquals(expected: Date, actual: Date) {
        Assert.assertEquals(expected.time / 1000, actual.time / 1000)
    }

    companion object {
        val priceList: PriceList by lazy {
            Helper.getSP500TestData()
        }

        val size: Int by lazy {
            priceList.size
        }
    }
}
